app/services/force/soql/listing_service.rb

```diff
@@ -14,6 +14,54 @@ def lease_up_listings
           WHERE Status__c = 'Lease Up' AND Is_Applicant_List_for_Leaseup__c = FALSE
         )))
       end
+
+      def pre_lottery_listings
+        # Show only listings that are not present in the lease up tab.
+        parsed_index_query(%(
+          SELECT #{query_fields(:index)} FROM Listing__c
+          WHERE Status__c != 'Lease Up' OR Is_Applicant_List_for_Leaseup__c = TRUE
```

Comment:
this is the only change here, the rest of it is copied over from the other listing service file